tests/console/kdump_and_crash.pm

```diff
@@ -46,10 +46,18 @@ sub run {
     # all but PPC64LE arch's vmlinux images are gzipped
     my $suffix = get_var('OFW') ? '' : '.gz';
     assert_script_run 'find /var/crash/';
+
     my $crash_cmd = "echo exit | crash `ls -1t /var/crash/*/vmcore | head -n1` /boot/vmlinux-`uname -r`$suffix";
-    assert_script_run "$crash_cmd", 600;
-    validate_script_output "$crash_cmd", sub { m/PANIC/ }, 600;
+    if (script_run($crash_cmd)) {
+        record_soft_failure 'boo#1090659 - crash: invalid kernel virtual address: xxxxxxxxx  type: yyyyyyyy';
+        script_run 'ls -lah /boot/';
+        script_run 'tar -cvf /tmp/crash_saved.tar /var/crash/*';
+        upload_logs '/tmp/crash_saved.tar' if is_sle('15+');
```

Comment:
why record the soft failure and save the tar file but not upload it on non-sle-15+?